src/ucp/core/ucp_context.c

```diff
@@ -57,6 +57,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "Threshold for switching from eager to rendezvous protocol",
    ucs_offsetof(ucp_config_t, rndv_thresh), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"LOG_DATA", "0",
+   "How much of packet payload to dump in debug mode (0 - disable).",
```

Comment:
"The size (in bytes) of the log buffer that is used for a logging of the network packet information (headers, payload, etc.). The information that does not fit into the log buffer will be truncated."